From f2b0819bb3aeff62519eda2a3a762a3716b5088a Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Thu, 30 Nov 2017 18:41:46 +0000
Subject: [PATCH] Update default crio.conf file for Clear Linux

Update paths to conmon and cni plugins to reflect where they are
intended to be used from Clear Linux.
---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index 82a35980..33b84ced 100644
--- a/Makefile
+++ b/Makefile
@@ -92,6 +92,8 @@ kpod: .gopathok $(shell hack/find-godeps.sh $(GOPKGDIR) cmd/kpod $(PROJECT))
 
 crio.conf: crio
 	./crio --config="" config --default > crio.conf
+	sed -i 's|^\(conmon = \).*|\1"/usr/libexec/crio/conmon"|' crio.conf
+	sed -i 's|^\(plugin_dir = \).*|\1"/etc/cni/bin/"|' crio.conf
 
 clean:
 ifneq ($(GOPATH),)
-- 
2.15.0

